src/libraries/sendtohelix.proj

```diff
@@ -4,6 +4,9 @@
     <HelixBuild Condition="'$(HelixBuild)' == ''">$(BUILD_BUILDNUMBER)</HelixBuild>
     <HelixBuild Condition="'$(HelixBuild)' == ''">default</HelixBuild>
 
+    <!-- To enable the interpreter for mono, we need to pass an env switch -->
+    <MonoEnvOptions Condition="'$(MonoEnvOptions)' == '' and '$(MonoEnableInterpreter)' == 'true'">--interpreter</MonoEnvOptions>
```

Comment:
Should this instead be set in a common place instead of duplicating it in `tests.props` and here?